getting-started/deploying-to-openshift.adoc

```diff
@@ -0,0 +1,235 @@
+= Deploying a BAMOE application to OpenShift using Quarkus
+include::../partials/attributes.adoc[]
+
+This guide provides instructions for deploying a business application to OpenShift using three different methods. Ensure that the following prerequisites are in place before proceeding.
+
+== Prerequisites
+
+* https://docs.openshift.com/container-platform/4.16/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[OpenShift CLI (`oc`)]
+* https://docs.docker.com/manuals/[Docker]
+
+== Deployment Options
+
+=== 1. Build and Deploy on OpenShift
```

Comment:
I don't think we should include the number in the heading for these three methods.